gammapy/scripts/image_pipe.py

```diff
@@ -64,9 +66,34 @@ def make_counts(self, obs_id):
         obs = self.data_store.obs(obs_id=obs_id)
         events = obs.events.select_energy(self.energy_band)
         events = events.select_offset(self.offset_band)
-        counts_map.fill(events=events)
-        counts_map.data = counts_map.data.value
-        self.maps["counts"] = counts_map
+        if (len(events) != 0):
+            counts_map.fill(events=events)
+            counts_map.data = counts_map.data.value
+            self.maps["counts"] = counts_map
+        else:
+            self.maps["counts"] = counts_map
+
+    def _has_counts(self, obs_id):
```

Comment:
How about this?

Change the no counts statement to a counts < min_counts statement where the user can set min_counts (maybe use 1 or 5 by default), change `_has_counts` to `_low_counts` and emit a log warning there:
```
log.warning('Skipping obs OBS_ID because it only has N_COUNTS counts.')
```

Then we can think more about how to handle very low counts data later.

OK?